### ISSUE TITLE
is correct ?

### ISSUE THREAD

**Issue by PsycoYT** [CigarProject/Cigar#468]
Hello , happy New Year..
I have created SetClan, to give a text just like a name 

I just have to define 'clan'

 ```
 for (var char, name = "";;) { // nick name
                char = view.getUint16(offset, true);
                offset += 2;
                if (0 == char) break;
                name += String.fromCharCode(char);
            }
```

Would be like: 

 ```
 for (var char, name, clan = "";;) { // nick name
                char = view.getUint16(offset, true);
                offset += 2;
                if (0 == char) break;
                name += String.fromCharCode(char);
                clan += String.fromCharCode(char);
            }
```

**Or is there a mistake? Forgive me, I'm learning javascript.**

**Comment by HAlexTM** [comment]
You need server side modifications, setNick function modifications, draw modifications

TL;DR it won't work.

**Comment by PsycoYT** [comment]
you can help me ? i actually use multiogar-edited please :)

**Comment by HAlexTM** [comment]
I can't, I’m really busy

**Comment by PsycoYT** [comment]
oh :( 
Could you give me some of your contact and then delete it quickly please?

**Comment by HAlexTM** [comment]
I don’t have any contact that knows JavaScript :)

**Comment by PsycoYT** [comment]
ANY1 help me?

**Comment by Luka967** [comment]
If you want only your clan tag to be visible, that's easy peasy. Put a text box or extract it from nick name and just draw it on cells if it's about 20 lines of code server-side and 30 lines client-side. (And no, I didn't actually make it)

**Comment by PsycoYT** [comment]
luka967 please help meee

**Comment by PsycoYT** [comment]
for (var char, name = "";;) { // nick name
                char = view.getUint16(offset, true);
                offset += 2;
                if (0 == char) break;
                name += String.fromCharCode(char);
			
            }


how to add clan ? i already edit the server I can only add that.

**Comment by PsycoYT** [comment]
@Luka967  @HAlexTM  


**if i add this:**

```
for (var char, name, clan = "";;) { // nick name
                char = view.getUint16(offset, true);
                offset += 2;
                if (0 == char) break;
                name += String.fromCharCode(char);
				clan += String.fromCharCode(char);
            }
```


**give me that error..**

![image](https://cloud.githubusercontent.com/assets/24230169/21539304/561f5ee2-cd84-11e6-8b78-387a36412100.png)

**Comment by HAlexTM** [comment]
Me:

> You need server side modifications, setNick function modifications, draw modifications
>
> TL;DR it won't work.

@Luka967:

> If you want only your clan tag to be visible, that's easy peasy. Put a text box or extract it from nick name and just draw it on cells.
If you want to see tags from other players, it needs server modifications. Needs about 20 lines of code server-side and 30 lines client-side. (And no, I didn't actually make it)

**Comment by PsycoYT** [comment]
![image](https://cloud.githubusercontent.com/assets/24230169/21539451/cb64993c-cd85-11e6-9111-1a66b2d5c710.png)



**Show me that mistake with that code:**

```
 for (var char, name, clan = '';;) { // nick name
                char = view.getUint16(offset, true);
                offset += 2;
                if (0 == char) break;
                name += String.fromCharCode(char);
				clan += String.fromCharCode(char);
            }
			
			var name = name + String.fromCharCode(char);
			
```

**Comment by PsycoYT** [comment]
@HAlexTM  you can help me?

**Comment by PsycoYT** [comment]
what is the error

**Comment by PsycoYT** [comment]
the console not show errors

**Comment by HAlexTM** [comment]
We can't help you, what language should I use to say that?

**Comment by PsycoYT** [comment]
OKK thanks.

**Comment by MastaCoder** [comment]
Why send those with packets? You can have it to send the nickname like so: [clan] user. No changes on the server needed, just on the client have it check for [], if it exists then split the text and have it rendered above the user. @PsycoYT

**Comment by RelTakeover** [comment]
@mastacoder help me please
